src/reports.coffee

```diff
@@ -169,13 +172,13 @@ htmlTemplate = (report) ->
   for data in report.data
     do (data) ->
       text += "<tr><td><i>#{data.channel.name}</i></td>"
-      text += "<td> #{ data.data[0].load } transactions </td>"
-      text += "<td> #{ data.data[0].avgResp } </td>"
-      text += "<td> #{ data.statusData[0].failed }  </td>"
-      text += "<td> #{ data.statusData[0].successful }  </td>"
-      text += "<td> #{ data.statusData[0].processing }  </td>"
-      text += "<td> #{ data.statusData[0].completed }  </td>"
-      text += "<td> #{ data.statusData[0].completedWErrors } </td></tr>"
+      text += "<td> #{ if data.data[0].load then data.data[0].load else 0 } transactions </td>"
+      text += "<td> #{ if data.data[0].avgResp then data.data[0].avgResp else 0 } </td>"
+      text += "<td> #{ if data.statusData[0].failed then data.statusData[0].failed else 0 }  </td>"
+      text += "<td> #{ if data.statusData[0].successful then data.statusData[0].successful else 0 }  </td>"
+      text += "<td> #{ if data.statusData[0].processing then data.statusData[0].processing else 0 }  </td>"
+      text += "<td> #{ if data.statusData[0].completed then data.statusData[0].completed else 0 }  </td>"
+      text += "<td> #{ if data.statusData[0].completedWErrors then data.statusData[0].completedWErrors else 0 } </td></tr>"
```

Comment:
@debonair this doesn't work, are you testing these before asking me to review it? This has the same problem that we were originally trying to solve. You null checks are too specific, you need to be testing data.data[0] and data.statusData[0] if they are null. You could use the existential operator if you want to check both. Something like: `data.data[0]?.load?` data.data[0] doesn't get checked.